Clarify that changing secrets.json requires reloading the secrets (#313)
bitwarden/contributing-docs: docs/getting-started/server/database/ef/index.mdx

</TabItem>
</Tabs>

:::note

> After making changes to your `secrets.json` file, remember to run `pwsh setup_secrets.ps1 -clear` to

Reviewer: ```suggestion
After making changes to your `secrets.json` file, remember to run `pwsh setup_secrets.ps1 -clear` so
```

> make sure the changes take effect.

Reviewer: ```suggestion
that the changes take effect.
```

> After making changes to your `secrets.json` file, remember to run `pwsh setup_secrets.ps1 -clear` to
> make sure the changes are incorporated.

Reviewer 2: Good tip! A slight nit / suggestion on wording:

```suggestion
After making changes to your `secrets.json` file, remember to run `pwsh setup_secrets.ps1 -clear` to
make sure the changes take effect.
```

Author: Good call!
